UI/importers/importers.cpp

```diff
@@ -0,0 +1,102 @@
+/******************************************************************************
+    Copyright (C) 2019 by Dillon Pentz <dillon@vodbox.io>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "importers.hpp"
+
+using namespace std;
+using namespace json11;
+
+vector<Importer *> importers;
```

Comment:
You should be using
```cpp
#include <memory>

// [...]

vector<unique_ptr<Importer>> importers;
```

Your pointers here are just unnecessarily leaked memory at this point.  Which I mean although will be automatically be cleaned up by the system when the program fully exits, but is still is not good practice.  Always avoid pointers.